### PR TITLE
Add LVM pool dashbaord

### PR DESCRIPTION
**Pull request by Ab-hishek**
This PR adds LVM pool dashboard
## Screenshots fo the dashboard:
Screenshot from 2021-06-30 10-34-31

Screenshot from 2021-06-30 10-34-47

Screenshot from 2021-06-30 10-35-01

Screenshot from 2021-06-30 10-35-13

Screenshot from 2021-06-30 10-35-31

Screenshot from 2021-06-30 10-35-45

Signed-off-by: Abhishek Agarwal <abhishek.agarwal@mayadata.io>

### CODE REVIEW

File: deploy/charts/openebs-monitoring/values.yaml
Comment (rajaSahil) [inline deploy/charts/openebs-monitoring/values.yaml]: Can we rename the directory to `lvm-LocalPV`?

File: deploy/charts/openebs-monitoring/dashboards/LVM/lvm-pool.json
Comment (rajaSahil) [inline deploy/charts/openebs-monitoring/dashboards/LVM/lvm-pool.json]: Make `multi` select option for dropdown as true. It can enable users to select multiple instances.

File: deploy/charts/openebs-monitoring/dashboards/LVM/lvm-pool.json
Comment (rajaSahil) [inline deploy/charts/openebs-monitoring/dashboards/LVM/lvm-pool.json]: Remove these values.

File: deploy/charts/openebs-monitoring/values.yaml
Comment (Ab-hishek) [reply deploy/charts/openebs-monitoring/values.yaml:191]: Changing it to LocalPV-lvm. We can add other flavours also under LocalPv-<localpv flavours> dir.

**Comment by avishnu** [comment]
In the thin-pool stats dashboard, can the health status have a non-empty value when its healthy?

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md]: You seem  to have checked-in a merge-diff file, ptal.

File: deploy/charts/openebs-monitoring/values.yaml
Comment (avishnu) [inline deploy/charts/openebs-monitoring/values.yaml]: LVMLocalPV or lvmLocalPV

Comment (avishnu) [reply deploy/charts/openebs-monitoring/values.yaml]: I think in the mixin PR, we have decided to follow the camel-case approach.

**Comment by Ab-hishek** [comment]
> In the thin-pool stats dashboard, can the health status have a non-empty value when its healthy?

I think yes. Because currently the pool in my cluster is healthy and the value is "". So, I have mapped this empty string value to healthy string.

**Review by avishnu — APPROVED** [review]
lgtm, will merge once @rajaSahil approves and clarifies on the lvmLocalPV variable naming.

File: deploy/charts/openebs-monitoring/values.yaml
Comment (rajaSahil) [reply deploy/charts/openebs-monitoring/values.yaml]: Yes @avishnu . We have decided to follow the camel-case approach for configuration. 
This is the directory name that we use to store dashboards json.  We can rename the directory to lvmLocalPV.

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:119]: jiva

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:120]: localPV

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:123]: cStor

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:145]: lvmLocalPV and above as well

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:146]: deviceLocalPV

File: deploy/charts/openebs-monitoring/README.md
Comment (avishnu) [inline deploy/charts/openebs-monitoring/README.md:151]: cStor

**Comment by avishnu** [comment]
Feel there are inconsistencies, can be either fixed in this PR or taken up in a separate PR as well.

**Review by kmova — APPROVED** [review]

**Review by rajaSahil — APPROVED** [review]

**Comment by Ab-hishek** [comment]
We can take these configuration changes in the next PR.

**Review by avishnu — APPROVED** [review]
Lgtm

**Pull request merged**
